packages/kbn-discover-utils/src/index.ts

```diff
@@ -9,3 +9,4 @@
 export * from './constants';
 export * from './hooks';
 export * from './utils';
+export * from './constants';
```

Comment:
Seems like this was added twice?
```suggestion
```

Reply:
Fixed in e23a04fe3adc4981b7f73da2962caae46e84e74e.